apps/site/assets/ts/hooks/useStop.ts

```diff
@@ -13,4 +15,15 @@ const useStop = (stopId: string): FetchState<Stop> => {
   return { status: FetchStatus.Data, data };
 };
 
-export default useStop;
+const useFacilitiesByStop = (stopId: string): FetchState<Facility[]> => {
+  const { data, error } = useSWR<Facility[]>(
+    `/api/facilities/${stopId}`,
```

Comment:
nit-pick:  I think the url should contain the word stop, since that is the id we are using to fetch the information, and the controller this function is in.  Without this context the current url would look like `/api/facilities/342` which would make me think we are grabbing a specific facility.
This end point should be something more like: `/api/stop/{stopId}/facilities`
That seems more to indicate this endpoint returns all the facilities for a given stop.